docs/can-guides/commitment/recipes/file-navigator/file-navigator-advanced.md

```diff
@@ -287,81 +286,61 @@ In this section, we’ll list the files and folders within the root folder.
 
 ### The solution
 
-The following uses `entitiesPromise` to write `<div class="loading">Loading</div>` while
-the promise is pending, and then writes out an `<li>` for each entity in the resolved `entitiesPromise`:
-
-```html
-<script type="text/stache" id="app-template">
-  <span>{{folder.name}}</span>
-  {{#if(entitiesPromise.isPending)}}
-    <div class="loading">Loading</div>
-  {{else}}
-    <ul>
-      {{#for(entity of entitiesPromise.value)}}
-        <li class="{{entity.type}} {{#if(entity.hasChildren)}}hasChildren{{/if}}">
-          {{#eq(entity.type, 'file')}}
-            📝 <span>{{entity.name}}</span>
-          {{else}}
-            📁 <span>{{entity.name}}</span>
-          {{/eq}}
-        </li>
-      {{/for}}
-    </ul>
-  {{/if}}
-</script>
-```
-@highlight 3-17
+Update the __JavaScript__ tab to:
 
-The following adds an `entitiesPromise` to data passed to the template.  `entitiesPromise`
+- Add `entitiesPromise` property to the `ViewModel`.  `entitiesPromise`
 will contain the files and folders that are directly within the root folder.
 
+- Use `entitiesPromise` to write `<div class="loading">Loading</div>` while
+the promise is pending, and then writes out an `<li>` for each entity in the resolved `entitiesPromise`
+
 @sourceref ./file-navigator-advanced-6.js
-@highlight 79,only
+@highlight 74-88,92-96,only
 
-## Toggle children with a ViewModel
+## Manage `<a-folder>` custom element behavior
 
 ### The problem
 
-We want to hide the root folder’s children until the root folder is clicked on.  An subsequent
-clicks on the root folder’s name should toggle if the children are displayed.
+Now we want to make all the folders able to open and close.
 
 ### Things to know
 
 - CanJS uses [guides/technicalViewModels#MaintainableMVVM ViewModels] to manage the behavior
```

Comment:
This link seems incorrect.

Reply:
Yeah it is from the current guide. it is fixed now.